interface/usergroup/usergroup_admin_add.php

```diff
@@ -645,6 +655,52 @@ function authorized_clicked() {
 
 });
 </script>
+
+<!-- Password Strength Meter JavaScript -->
+<script>
+function checkPasswordStrength() {
+    console.log("checkPasswordStrength called");
+    var number = /([0-9])/;
+    var alphabets = /([a-zA-Z])/;
```

Comment:
Your password strength algorithm is not multi-lingual.  Your regex assumes that everyone using the system will create passwords with only ASCII characters which is not correct as I'm pretty sure OpenEMR passwords can support unicode characters.  That's correct isn't it @bradymiller?

Reply:
Resolved in my latest commit.